demos/demo-react-ts/src/components/App.tsx

```diff
@@ -35,6 +35,21 @@ export const INBOUND_SCREENS = [
   CallEndedScreen,
 ];
 
+const ALERT_CONTENT = (
+  <p>
+    Open your console to see the incoming and outgoing messages with HubSpot.
+    Inbound call support is in beta. For more info, refer{" "}
```

Comment:
The alert content is causing a scrollbar to show up on the keypad screen. Should we keep the message concise here, i.e. "Open your console to see the [incoming (beta)](https://developers.hubspot.com/docs/api/crm/extensions/receive-calls-in-hubspot-when-using-calling-apps) and outgoing messages with HubSpot." 
<img width="400" alt="image" src="https://github.com/HubSpot/calling-extensions-sdk/assets/23175119/1a7f7254-7daf-45b0-b08d-f25048ea8f88">

Reply:
Thanks. I think your suggested text works